src/main/java/com/ibm/watson/developer_cloud/personality_insights/v2/model/ContentItem.java

```diff
@@ -25,77 +25,20 @@
 public class ContentItem extends GenericModel {
   private String charset;
   private String content;
-  private String contenttype;
+  private String contentType;
```

Comment:
I like that you renamed the Java fields to use proper capitalization. Our JSON deserialization library `Gson` is strict about naming though. Because of that you need to add a `@SerializedName("contenttype")` to the renamed fields to match the JSON keys (see [Personality Insights](https://watson-api-explorer.mybluemix.net/apis/personality-insights-v2#!/personality-insights/profile) under *body*). Elsewise GSON will miss these fields during parsing.